tracekit.js

```diff
@@ -144,7 +144,7 @@ TraceKit.report = (function reportModuleWrapper() {
 
     /**
      * Dispatch stack information to all handlers.
-     * @param {Object.<string, *>} stack
+     * @param {TraceKit.StackTrace} stack
      * @memberof TraceKit.report
```

Comment:
It occurs to me that we're missing a parameter here. I'll fix that in a separate PR since it doesn't have anything to do with stack traces.